### DISCUSSION TITLE
Documentation question regarding "Include Restarting"

### DISCUSSION THREAD

**Discussion by cwallwey** [containrrr/watchtower#1399] [Q&A] [answered]
Regarding the "Include Restarting" argument, the documentation says "Will also include restarting containers". Does this mean that containers will be restarted after the update, or that containers that are restarted while Watchtower is checking them will also be checked? I assume it's the latter, but to be sure. Thanks!

**Comment by piksel** [comment, marked as answer]
Yeah, normally only containers in the `running` state are considered, but if they were in a faulty state they might be crashing and restarting repeatedly and would never be updated. This would include such containers in the check/updates.

**Reply by piksel** [reply]
I do understand what you are referring to though.
`Will also include restarting containers` could just as well mean, "the operation performed on the containers will also include restarting them". Perhaps the wording ought to be altered slightly...

**Reply by cwallwey** [reply]
Thanks for clarification. Appreciate it.
